submissions/Zakkarat/memory-pair-game/app.js

```diff
@@ -0,0 +1,66 @@
+const cards = document.getElementsByClassName('flipper');
+const cont = document.getElementsByClassName('container')[0];
```

Comment:
you saved 5 letters and make you variables mysterious, imo. why don't you like `container` :)

Reply:
Done!